### PR TITLE
hotfix(docker): add license header to xml fixtures

### PR DESCRIPTION
**Pull request by duncdrum**
`mvn license:format` we could probably adapt the shell script recommendations to also cover `.bats` files

### CODE REVIEW

**Review by dizzzz — APPROVED** [review]

**Comment by duncdrum** [comment]
CI failures come from daef2c5909def3005ea589756a8f97d53d647af1 and are unrelated to this PR. The docker changes are working and tests are passing, also added a check to avoid future regressions.

**Comment by line-o** [comment]
@duncdrum Care to elaborate what exactly causes the CI failures, since you seem to be in the know?

**Comment by line-o** [comment]
We need to wait for the rollback to happen first before this can be merged.
Thanks for fixing this.

**Review by line-o — APPROVED** [review]
Will merge after master is rolled back

**Comment by line-o** [comment]
I think it is best to wait for further info before we merge anything into master.

**Pull request merged**
